submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/service/DoiService.kt

```diff
@@ -55,7 +55,7 @@ class DoiService(
     private fun registerDoi(accNo: String, rqt: SubmitRequest): String {
         val sub = rqt.submission
         val timestamp = Instant.now().epochSecond.toString()
-        val title = requireNotNull(sub.title) { throw MissingTitleException() }
+        val title = rqt.submission.find(TITLE) ?: rqt.submission.section.find(TITLE) ?: throw MissingTitleException()
```

Comment:
in the first call we search in case sensite and in the second it is not. Are these the expectations?

Reply:
Nice catch